docs/server-side/v2/how-to-guides/configure-webhooks/webhook.md

```diff
@@ -1213,28 +1213,40 @@ This event will be sent when HLS recordings are successful and uploaded to the s
         "hls_vod_recording_size": 10024,
         "recording_single_files": [
             {
+                "asset_id": "<asset_id_of_the_recording>",
```

Comment:
put a sample asset id , instead of the placeholder <asset_id_of_the_recording>

Reply:
Done